README.md

```diff
@@ -35,6 +35,18 @@ For example:
 ./generate-manifest-collector.sh --mode dev --port 4739 --proto tcp > ../build/yamls/ipfix-collector.yaml
 ```
 
+### Start a Kafka instance
+To start a Kafka server with minimum Kubernetes setup, run
+```shell
+kubectl apply -f ./build/yamls/kafka-server.yaml -n [NAMESPACE]
+``` 
+Broker address will be `kafka-service:9092`.
+Then we can connect a Kafka consumer to this server by running: 
+```shell
+kubectl create deployment --image=antrea/kafka-consumer consumer -n [NAMESPACE]
+``` 
```

Comment:
We should have a single manifest for kafka broker and consumer if possible to keep the deployment simple. We already have a workflow to upload the cosumer image.

I think it is ok to pull the image from the confluent repo in the case of Kafka borker. This is similar to ELK flow collector. However, there may be an issue in Antrea e2e tests if there are any pull errors from docker registry. It is better to get a copy and upload it to the vmware harbor registry. 
Maybe we should have a check on the version and upload the image through a daily job. What do you think?

Reply:
Yes it makes sense to keep image copies at our side. Will create a job in Antrea and update our yaml accordingly.

Reply:
It doesn't have to be Antrea. We can do it in go-ipfix? Do you see an issue? We have access to Antrea repo on harbor registry.

Reply:
Oh right. we can do it here.